mapa.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="pl">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Pikna Mapa</title>
+  <style>
+    /* TODO: przenieść to do osobnego pliku */
+    #zawartosc, #mapa {
+      margin: 0 auto;
+    }
+    #tytul {
+      text-align: center;
+    }
+  </style>
+</head>
```

Comment:
Nie wiem czy to wszystko jest potrzebne, jeżeli to jest tylko frame... Jakieś źródło?